.github/workflows/docs-build-push.yml

```
@@ -52,6 +57,7 @@ jobs:
       docs_build_path: "./site"
       doc_type: "hugo"
       environment: ${{ inputs.environment }}
+      force_hugo_theme_version: ${{inputs.hugo_theme_override}}
```

nit: Is there a reason why spaces aren't included?

```suggestion
      force_hugo_theme_version: ${{ inputs.hugo_theme_override }}
```

---

Not intentional.